CONTRIBUTING.md

```diff
@@ -92,23 +92,23 @@ The commit message should be structured as follows:
 
 ---
 
-This repo uses automated tools to standardize the formatting of code, text files and commits.
+This repo uses automated tools to standardize the formatting of code, text files, and commits.
```

Comment:
I thought a comma before and is only added if it connects independent clauses 🤓 😅

Reply:
Yeah i checked a lot but that's the correct way lol